include/can/core/messages.hpp

```diff
@@ -20,7 +20,7 @@ using namespace can::ids;
 
 using stepper_timer_ticks = uint32_t;
 using brushed_timer_ticks = uint32_t;
-using um_per_tick = int32_t;
+using mm_per_tick = int32_t;
```

Comment:
wait shouldn't this stay as um_per_tick?

Reply:
velocity should be mm_per_tick